R/manual_layer_node.R

```diff
@@ -333,6 +333,9 @@ Node <- R6::R6Class(
 ##'
 ##' The task graph is implicitly defined by various \code{delayed} objects having others
 ##' in their argument lists.
+##
```

Comment:
This was missing from the roxygen block

Reply:
(If you meant to add to `roxygen2` processing then you probably wanted an aprostrophe behind the hashes too.  Then again it an empty line....)

Reply:
thanks @eddelbuettel ! :)